tests/conftest.py

```diff
@@ -57,17 +56,15 @@ async def mongo() -> AsyncGenerator[Mongo, None]:
                      authSource="admin")
     await fc_mongo.create_indexes()
 
-    try:
-        # provide the client as the fixture
-        yield fc_mongo
+    # provide the client as the fixture
+    yield fc_mongo
 
     # -------------------------------------------------------------------------
     # NOTE: *Your Unit Test Function Runs Here*
     # -------------------------------------------------------------------------
 
     # teardown_function
-    finally:
-        fc_mongo.close_me.close()
+    fc_mongo.close_me.close()
 
```

Comment:
I think this change is fine because a `fixture` uis technically a context manager. I'm not entirely sure `fc_mongo.close_me.close()` will execute if the consumer (unit test) raises an error. However, I'm not sure it matters in this case

Reply:
I tested it. If the unit test raises an Exception (and that's all my test did), pytest's test runner catches it, then comes back to tear down the fixture, so it gets closed. That made me realize the try-finally block wasn't really necessary.